project/GOVERNANCE.md

```diff
@@ -26,32 +26,55 @@ to appreciate the absence of bugs, the slow but steady improvement in stability,
 or the reliability of a release process. But those things distinguish a good
 project from a great one.
 
-### Adding maintainers
+## Reviewers
 
-Maintainers are first and foremost contributors who have shown their
-commitment to the long term success of a project. Contributors who want to
-become maintainers first demonstrate commitment to the project by contributing
-code, reviewing others' work, and triaging issues on a regular basis for at
-least three months.
+A reviewer is a maintainer within the project.
+They share in reviewing issues and pull requests and their LGTM counts towards the
+required LGTM count to merge a code change into the project.
```

Comment:
Do we need to explicitly define the "required LGTM count"?

Reply:
I'd leave that to the committers to decide in a follow up. You know my position on that has been to leave the proper number up to the discretion of the merging committer. A set value could be nice to put in branch protections though.